pages/db/[language]/[file]/text.tsx

```diff
@@ -29,21 +31,77 @@ export { getDbViewFileStaticPaths as getStaticPaths } from "utils/nextJsHelpers"
  *
  * @constructor
  */
+
+const Loading = () => (
```

Comment:
do we want to reuse this in other pages? Should it be defined outside of this file maybe?